src/main/resources/application-local.yml

```
@@ -29,3 +29,6 @@ spring:
       host: localhost
       port: 6379
 
+discord:
+  alert:
+    enabled: false
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)